src/test/java/uk/gov/hmcts/ccd/data/user/AttributeBasedJurisdictionsResolverTest.java

```diff
@@ -97,7 +111,134 @@ public void shouldReturnJurisdictionsForValidRoleAssignments() {
 
         List<String> result = jurisdictionsResolver.getJurisdictions();
         assertThat(result.size(), is(2));
-        assertTrue(result.contains(DIVORCE_JURISDICTION));
-        assertTrue(result.contains(PROBATE_JURISDICTION));
+        assertTrue(result.contains(DIVORCE_JURISDICTION.toLowerCase()));
+        assertTrue(result.contains(PROBATE_JURISDICTION.toLowerCase()));
+    }
+
+    @Test
+    void shouldReturnJurisdictionsOnlyWhereGrantIsNotBasic() {
+        given(idamUser.getId()).willReturn(USER_ID);
+        given(userRepository.getUser()).willReturn(idamUser);
+
+        given(roleAssignmentService.getRoleAssignments(USER_ID)).willReturn(
+            RoleAssignments.builder()
+                .roleAssignments(asList(
+                    RoleAssignment.builder()
+                        .grantType(GrantType.BASIC.name())
+                        .roleName(DIVORCE_SOLICITOR_ROLE)
+                        .actorId(USER_ID)
+                        .attributes(RoleAssignmentAttributes.builder()
+                            .jurisdiction(Optional.of(DIVORCE_JURISDICTION)).build()).build(),
+                    RoleAssignment.builder()
+                        .grantType(GrantType.CHALLENGED.name())
+                        .roleName(PROBATE_SOLICITOR_ROLE)
+                        .actorId(USER_ID)
+                        .attributes(RoleAssignmentAttributes.builder()
+                            .jurisdiction(Optional.of(PROBATE_JURISDICTION)).build()).build()
+                ))
+                .build()
+        );
+        List<String> result = jurisdictionsResolver.getJurisdictions();
+        assertThat(result.size(), is(1));
+        assertTrue(result.contains(PROBATE_JURISDICTION.toLowerCase()));
+    }
+
+    @Test
+    void shouldReturnJurisdictionsWhereNoJurisdictionProvided() {
```

Comment:
Think this one is testing when jurisdiction IS provided?
```suggestion
    void shouldReturnJurisdictionsWhereJurisdictionProvided() {
```
And can remove the first role assignment in this test data since no jurisdictions or case types is covered under a separate test where `caseDefinitionRepository.getAllJurisdictionsFromDefinitionStore()` had to be mocked

Reply:
done

Reply:
done